common/views/components/IIIFViewer/parts/GridViewer.js

```diff
@@ -103,27 +109,48 @@ const GridViewer = ({
   setActiveIndex,
   canvases,
   isFullscreen,
+  viewerRef,
 }: Props) => {
+  const { infoBanner } = useContext(GlobalAlertContext);
   const [newScrollOffset, setNewScrollOffset] = useState(0);
   const scrollVelocity = useScrollVelocity(newScrollOffset);
   const itemWidth = 250;
   const columnCount = Math.round(gridWidth / itemWidth);
   const columnWidth = gridWidth / columnCount;
   const grid = useRef();
-
   useEffect(() => {
     const rowIndex = Math.floor(activeIndex / columnCount);
     grid &&
       grid.current &&
       grid.current.scrollToItem({ align: 'start', rowIndex });
   }, [activeIndex]);
+  useEffect(() => {
+    setTimeout(() => {
+      window.scrollTo(0, 0);
```

Comment:
this part is very adhoc, but its difficult to do this other ways.

Reply:
Could you leave a comment as to why this is like this, maybe a link to an example? I can't tell from just looking at it, and I might remove it as a future develop unless I knew why it was there.

Reply:
yep, will do